timetracker/entries/templates/clients.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <ul class="nav">
+        <li><a href="/entries/">Entries</a></li>
+        <li><a href="/clients/" class="active">Clients</a></li>
+        <li><a href="/projects/">Projects</a></li>
+    </ul>
+
+    <h2>Clients</h2>
+    <ul>
+        {% for client in client_list %}
+            <li><a href="/client?client_id={{ client.id }}">{{ client.name }}</a></li>
```

Comment:
Ideally this would use {% url %} to look up the link, instead of hard coding it.  This way, if you re-map your urls later in the project, you don't have to update all of your templates.

Also, normally we'd pass the client id as part of the URL, so the url pattern can parse it out.  QueryStrings are more often used to filter a list of results than to specify a single item.